scout/artifactory.md

```diff
@@ -20,13 +20,25 @@ title: Artifactory integration
 Integrating Docker Scout with JFrog Artifactory lets you run image analysis
 automatically on images in your Artifactory registries.
 
-This integration is made possible by a monitoring agent. The agent is a
+## Local image analysis
+
+You can analyze Artifactory images for vulnerabilities locally using Docker Desktop or the Docker CLI. You first need to authenticate with JFrog Artifactory using the `[Docker login](/engine/reference/commandline/login/)` command. For example:
+
+```bash
+docker login {REPOSITORY_URL}.jfrog.io
+```
+
+You can find the credentials for your Artifactory repository by selecting it in the Artifactory UI and then the **Set Me Up** button.
+
+## Remote image analysis
+
+To automatically analyze images and containers running in remote environments you need to deploy the Docker Scout Artifactory agent. The agent is a
```

Comment:
Technically we aren't analyzing containers, just images.

Reply:
```suggestion
To automatically analyze images running in remote environments you need to deploy the Docker Scout Artifactory agent. The agent is a
```